src/Microsoft.AspNet.OData.Shared/Query/Expressions/DynamicTypeWrapper.cs

```diff
@@ -35,7 +40,7 @@ public bool TryGetPropertyValue(string propertyName, out object value)
     [JsonConverter(typeof(DynamicTypeWrapperConverter))]
     internal class GroupByWrapper : DynamicTypeWrapper
     {
-        private Dictionary<string, object> _values;
+        protected Dictionary<string, object> _values;
```

Comment:
>_values [](start = 45, length = 7)

If this is protected (and not private) it should follow standard (i.e., pascal) naming conventions.  Really, fields should not be protected (or public); the preferred mechanism is to provide getters/setters, or other methods for accessing/mutating.